charts/prometheus-node-exporter/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.0.1
 description: A Helm chart for prometheus node-exporter
 name: prometheus-node-exporter
-version: 1.12.0
+version: 1.13.0
```

Comment:
Hi @cgroschupp there's already a `1.13.0` [refer here](https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-node-exporter/Chart.yaml#L5). Please bump the chart version.

Reply:
done